Bubble_Sort/bubble_sort.ts

```diff
@@ -0,0 +1,44 @@
+//Bubble Sort In Typescript
+
+function bubble(array : number[]) : number[]
+{
+ console.log("Original Array : " + array); // print original array
```

Comment:
Please indent all the lines inside this function by 4 spaces.

Reply:
done!